### ISSUE TITLE
Stgcn cannot run under Pytorch using dataset ntu60_xsub.

### ISSUE THREAD

**Issue by pdx1989** [open-mmlab/mmaction2#2141]
My running cmd is:
python tools/train.py configs/skeleton/stgcn/stgcn_80e_ntu60_xsub_keypoint_3d.py --validate --seed 0 --deterministic --gpus 1
And,
mmcv version: 1.7.0
PyTorch version: 1.10.2+cu102
TorchVision: 0.4.0a0
OpenCV: 4.6.0
MMAction2: 0.24.1+1baad1a

While the execution result is:
<img width="428" alt="企业微信截图_16710937551394" src="https://user-images.githubusercontent.com/4810833/207820303-c782db9f-d6ec-4d0a-9986-68c043aec032.png">

It seems that bn size is different between self.graph.A and self.running_means
Could anyone help me? Thanks!

**Comment by Dai-Wenxun** [comment]
Did you use 2d keypoint skeleton data for training? This config is for 3d keypoint skeleton data, which has 25 joints.

**Comment by Dai-Wenxun** [comment]
The dataset you used is for 2d keypoint. Please refer to this [tutorial](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton#convert-the-ntu-rgbd-raw-skeleton-data-to-our-format-only-applicable-to-gcn-backbones) for more details about how to prepare 3d keypoint dataset.

**Comment by pdx1989** [comment]
> The dataset you used is for 2d keypoint. Please refer to this [tutorial](https://github.com/open-mmlab/mmaction2/tree/master/tools/data/skeleton#convert-the-ntu-rgbd-raw-skeleton-data-to-our-format-only-applicable-to-gcn-backbones) for more details about how to prepare 3d keypoint dataset.

I have tried gen_ntu_rgbd_raw.py with the raw data "NTU RGB+D - RGB Videos", and throws the following errors. Do I need to run ntu_pose_extraction.py at first? What other possible mistakes could I have?

<img width="702" alt="image_dataset" src="https://user-images.githubusercontent.com/4810833/208877075-a38fb98a-bb11-4db5-a60d-4b1b3462518a.png">

**Comment by Dai-Wenxun** [comment]
You just need to download the preprocessed 3d skeleton data. [link](https://github.com/shahroudy/NTURGB-D) 
![image](https://user-images.githubusercontent.com/58767402/208881963-cbb3f048-f778-4892-b721-3f8d868fce53.png)

Then, put all the samples (e.g., S001C002P003R002A013.skeleton) into the folder - `data/ntu/nturgb+d_skeletons_60/` and run the `gen_ntu_rgbd_raw.py`.
